Clean up & New pages

AsherSpurr opened the pull request:
# Pull Request
 Tea education pages, clean up of styling, fetch function for single tea
## Description
Adds two new components, TeaEd and TeaArticle. Adds fetch function for a single tea. Formats document and removed unused comments
## Related Issues
Updates tea blends button to be blends. 
## Screenshots (if applicable)
![Screenshot 2024-04-06 at 6 41 50 PM](https://github.com/corysanders3/spilled/assets/144856487/f5745f6a-d698-4806-936b-40f15a30156b)
![Screenshot 2024-04-06 at 6 41 06 PM](https://github.com/corysanders3/spilled/assets/144856487/eaa35374-1555-4b88-917c-d870ecf62f89)

## Changes Made
**Additions:**
- TeaEd component - contains logic for its child TeaArticle
- Adds new interface for hardcoded Tea info/education
- dynamic routing to and from TeaArticle pages
- fetch function for single tea -- not integrated/unused

**Removals:**
- empty space
- unused comments
## Testing
No testing added for these pages yet
## Checklist
- [x] The code follows the project's coding standards.
- [ ] Unit tests have been added or updated to cover the changes.
- [x] Documentation has been updated to reflect the changes (if applicable).
- [x] The code compiles without errors.
- [ ] The changes have been tested locally and pass all relevant tests.
- [ ] All new and existing tests pass.
- [x] The pull request has been reviewed by at least one other contributor.
## Additional Information
**New components are completely unstyled**

AsherSpurr committed:
Merge branch 'style/cards' of github.com:corysanders3/spilled into chore/clean-up-as

AsherSpurr committed:
Chore: create hardcoded tea interface for Home page

AsherSpurr committed:
Hard coded Home Page Tea data

AsherSpurr committed:
Chore: change blend endpoint to blends

tednaphil committed:
Add flavor notes to cards

AsherSpurr committed:
Add teaEd component to display info about types

tednaphil committed:
Capitalize category headers

AsherSpurr committed:
Add routing for tea education

tednaphil committed:
Add placeholder for empty favs page

tednaphil committed:
Add home link to empty favs announcement

AsherSpurr committed:
Successful routing to education pages

AsherSpurr committed:
Add nav button to teas inside education

AsherSpurr committed:
Add navlink back home in education

AsherSpurr committed:
Add single tea fetch

AsherSpurr committed:
update fetch link

AsherSpurr committed:
remove spaced and format entire document

corysanders3 reviewed (approved):
Love all the tea hardcoded info!! Awesome detail. Really appreciate that.

tednaphil commented on src/components/home/Home.tsx:
```diff
@@ -1,54 +1,53 @@
       <div className="intro-wrapper">
         <article className="article-intro">
-          <img src={teas} />
+          <img src={teas} alt='Cupcake baking pan with different tea blends filling each cup' />
```
Yay for descriptive alt text!!

tednaphil commented on src/components/home/Home.tsx:
```diff
@@ -1,54 +1,53 @@
+            <p className="info-desc">
+              Did you know that all 'teas' originate from the leaves of the Camellia
+              Sinensis plant?(break this up) Any 'tea' that is not from this plant is actually an herbal blend! (break here) What makes each tea unique is
+              the creation process is goes through, including:
```
This is fabulously engaging content! Thank you for working on getting the intros together @AsherSpurr !

tednaphil commented on src/utils/interface.tsx:
```diff
@@ -19,4 +19,111 @@ export interface Tea {
+    climate: string
+}
+
+export const allHardcodedTeas: TeaHardcoded[] = [
```
WHOA! 💯

tednaphil reviewed (changes requested):
This all looks good but it seems like there are some merge conflicts to be resolved before it's able to be merged to main. If yall want to sift through the conflicts in the vscode merge editor by merging main into this branch beforehand, we can start with that Sunday morning!

tednaphil committed:
Merge branch 'main' into chore/clean-up-as

tednaphil committed:
Fix: blends path
>
>
Co-authored by: Asher Spurr <asher.spurr@gmail.com>
Co-authored by: Cory Sanders <corysanders3@yahoo.com >
Co-authored by: Tayla Phillips <richardstayla@gmail.com>

tednaphil committed:
Update card class

tednaphil committed:
Co-authored-by: Asher Spurr <AsherSpurr@users.noreply.github.com>
Co-authored-by: Cory Sanders <corysanders3@users.noreply.github.com>
